pwndbg/commands/leakfind.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Find a chain of leaks given some starting address.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+from queue import *
+
+import gdb
+
+import pwndbg.color.chain as C
+import pwndbg.color.memory as M
+import pwndbg.color.theme as theme
+import pwndbg.commands
+import pwndbg.vmmap
+
+config_arrow_right = theme.Parameter('chain-arrow-right', '—▸', 'right arrow of chain formatting')
+arrow_right = C.arrow(' %s ' % config_arrow_right)
```

Comment:
Note that it doesn't work well if the config/theme is applied after loading pwndbg - and this is what usually (always?) occurs:
```
pwndbg> set chain-arrow-right xxx
Set right arrow of chain formatting to 'xxx'
pwndbg> leakfind -o 100 $rsp
0x7fffffffe440+0x8 —▸ 0x7fffffffe6b5 [stack]
0x7fffffffe440+0x18 —▸ 0x7fffffffe6bd [stack]
```

We need to fix this.

Reply:
Also, the `chain-arrow-right` is already defined in another file and I believe we need to import it here instead of redefining it.

Reply:
Fixed. imported from pwndbg.chain